app/components/navs/nav-rail-common-item.jsx

```
@@ -87,8 +89,27 @@
+                setAnimationDirection('slide-left');
+            } else {
+                setAnimationDirection('slide-right');
+            }
+
+            setPrevPathname(pathname);
+        }
+    }, [pathname, prevPathname]);
+    
     return (
-        <div className={`nav-item w-full rounded-full cursor-pointer font-medium selected-${props.selected} transition-all duration-300 ease-in-out`} role='link'>
+        <div className={`nav-item w-full rounded-full cursor-pointer font-medium selected-${props.selected} transition-all duration-300 ease-in-out ${animationDirection}`} role='link'>
```

Biome (1.9.4)

[error] The HTML element with the interactive role 'link' is not focusable.

A non-interactive HTML element that is not focusable may not be reachable for users that rely on keyboard navigation, even with an added role like 'link'.
Add a tabIndex attribute to make this element focusable.

(lint/a11y/useFocusableInteractive)